src/components/Sum/Sum.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+export const Sum = ({ a = 0, b = 0 }) => {
```

Comment:
you can set default value using `defaultProps`
Also, add propTypes for the variables